bitrise.yml

```diff
@@ -2725,7 +2725,7 @@ workflows:
 
             brew install jq
 
-            resp=$(curl -X GET -s -H 'Accept: application/vnd.github.v3+json' -H "authorization: Bearer ${GITHUB_ACCESS_TOKEN}" https://api.github.com/repos/mozilla-mobile/firefox-ios/commits\?sha\=main\&since\=$YESTERDAY | jq -r '.[].commit.author | select(.name | contains("Firefox Test Engineering"))')
+            resp=$(curl -X GET -s -H 'Accept: application/vnd.github.v3+json' -H "authorization: Bearer ${GITHUB_ACCESS_TOKEN}" https://api.github.com/repos/mozilla-mobile/firefox-ios/commits\?sha\=main\&since\=$YESTERDAY | jq -r '.[].commit.message | select(contains("Auto Update Bitrise"))')
```

Comment:
Once we have mergify in place we can query by author again since it appears as the user who merged

Reply:
See as an example: 
`"commit": { "author": { "name": "isabelrios", "email": "isabelrios@users.noreply.github.com", "date": "2021-02-11T22:42:35Z" }, "committer": { "name": "mergify[bot]", "email": "37929162+mergify[bot]@users.noreply.github.com", "date": "2021-02-11T22:42:51Z" }, "message": "Update bitrise",`
The modification we would need: 
`| jq -e -r '.[].committer.login | select(contains("mergify[bot]"))`

Reply:
Let me know if you need more assistance with jq but this looks great.